modules/landing-page/landing-dashboard-overview.tsx

```
@@ -0,0 +1,346 @@
+                <Button size="sm" variant="ghost">
+                  <ChevronDown className="h-4 w-4" />
+                </Button>
+              </div>
+
+              <div className="space-y-2">
+                {todayTasks.map((task) => (
+                  <div
+                    key={task.id}
+                    className="group flex items-center space-x-3 rounded-md border border-gray-200 bg-white p-3 transition-colors hover:border-gray-300"
+                  >
+                    <button className="flex-shrink-0">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)